braviarc/braviarc.py

```diff
@@ -146,6 +146,23 @@ def send_command(self, command):
         """Sends a command to the TV."""
         self.send_req_ircc(self.get_command_code(command))
 
+    def get_source(self, source):
+        """Returns list of Sources"""
+        original_content_list = []
+        stIdx = 0
+        while True:
+            resp = self.bravia_req_json("sony/avContent",
+                                        self._jdata_build("getContentList", {"source": source, "stIdx": stIdx}))
+            if not resp.get('error'):
+                if len(resp.get('result')[0]) == 0:
+                    break
+                else:
+                    stIdx = resp.get('result')[0][-1]['index']+1
```

Comment:
why? a -1 index? could you please paste an example of data returned by your TV showing the -1 key just for clarification?